src/Php74/Php74.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Polyfill\Php74;
+
+/**
+ * @author Ion Bazan <ion.bazan@gmail.com>
+ *
+ * @internal
+ */
+final class Php74
+{
+    /**
+     * @param object $obj
+     *
+     * @return array
+     */
+    public static function get_mangled_object_vars($obj)
+    {
+        if (!$obj instanceof \ArrayIterator && !$obj instanceof \ArrayObject) {
+            $result = array();
+
+            foreach ((array) $obj as $key => $value) {
+                $result[(string) $key] = $value;
```

Comment:
isn't this exactly `return (array) $obj` ?

Reply:
This fixes a BC break introduced in PHP 7.2: https://3v4l.org/MuD5X

Reply:
@stof However I am not sure how exactly does it fix the issue - the casting seems optional here :man_shrugging: